corehq/apps/app_manager/const.py

```diff
@@ -97,3 +97,8 @@
 CALCULATED_SORT_FIELD_RX = r'^_cc_calculated_(\d+)$'
 
 CASE_LIST_FILTER_LOCATIONS_FIXTURE = 'Locations Fixture'
+
+MOBILE_UCR_V1_FIXTURE_IDENTIFIER = 'src="jr://fixture/commcare:reports'
+MOBILE_UCR_V1_FIXTURE_PATTERN = r'<.*src="jr://fixture/commcare:reports.*>'
+MOBILE_UCR_V1_REFERENCES_PATTERN = r"<.*instance\('reports'\)/reports/.*>"
+MOBILE_UCR_V1_ALL_REFERENCES = f"{MOBILE_UCR_V1_FIXTURE_PATTERN}|{MOBILE_UCR_V1_REFERENCES_PATTERN}"
```

Comment:
```suggestion
MOBILE_UCR_V1_ALL_REFERENCES = f"{MOBILE_UCR_V1_FIXTURE_PATTERN}|{MOBILE_UCR_V1_REFERENCES_PATTERN}"
# Pattern for references in case list/detail. 
# The below instance needs to be referred in a fixture setup and is then later used in Case List/Detail filters and columns.
V1_CASE_LIST_REFERENCES_PATTERN = r"instance\('commcare:reports'\)/reports/report\[@id='.*'\]/rows/row"
```

Reply:
Addressed in [0cf6206](https://github.com/dimagi/commcare-hq/pull/35168/commits/0cf6206de2829c5b157acdfaf062aef9fd662fdf).